app/queries/drug_stocks_query.rb

```diff
@@ -25,9 +26,9 @@ def drug_stocks_report
        patient_count_by_facility_id: patient_count_by_facility_id,
        patient_days_by_facility_id: patient_days_by_facility_id,
        drugs_in_stock_by_facility_id: drugs_in_stock_by_facility_id,
-       patient_count_by_block_id: patient_count_by_block_id,
-       patient_days_by_block_id: patient_days_by_block_id,
-       drugs_in_stock_by_block_id: drugs_in_stock_by_block_id,
+       patient_count_by_block_id: @include_block_report ? patient_count_by_block_id : {},
+       patient_days_by_block_id: @include_block_report ? patient_days_by_block_id : {},
+       drugs_in_stock_by_block_id: @include_block_report ? drugs_in_stock_by_block_id : {},
```

Comment:
Any reason to include these keys if `include_block_report` is false? Could we set to `nil` and `compact` at the end?

Reply:
TIL hashes can be compacted 🙌🏼